chromium_src/chrome/browser/ui/views/frame/browser_frame.h

```diff
@@ -6,13 +6,15 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_FRAME_BROWSER_FRAME_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_FRAME_BROWSER_FRAME_H_
 
-#define SelectNativeTheme         \
-  SelectNativeTheme_Unused() {}   \
+#include "ui/views/context_menu_controller.h"
```

Comment:
Is this include needed?

Reply:
Fixed in 655f791ee70bcdda512a6baca5fdc0634695eb8b.